1_ses_node_on_openstack/playbooks/play.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- hosts: all:!localhost
+- hosts: ses_nodes
```

Comment:
Where is this host group defined ?

Reply:
This group was indeed created into step 1, a little below.